### ISSUE TITLE
Add Bootstrap docs

### ISSUE THREAD

**Issue by pitaj** [Thibaut/devdocs#149]
Please add the Twitter Bootstrap docs found [here](http://getbootstrap.com/)

Really enjoying DevDocs.
Thanks!

**Comment by brutalcrozt** [comment]
It will be awesome if avaliable

**Comment by dandv** [comment]
+1

**Comment by Thibaut** [comment]
You can vote for Bootstrap here: https://trello.com/c/rmzn8RN0/38-bootstrap
The problem is that the docs are pretty much useless without the CSS and JS examples, which DevDocs doesn't support (because the docs are injected directly in the main frame, not in an iframe).

**Comment by dandv** [comment]
How about injecting the docs in a WebComponents, which prevents any CSS and JS bleed, and using Polymer as the polyfill if necessary (though most developers will use Chrome anyway to read devdocs)?

**Comment by Thibaut** [comment]
@dandv This would require too many changes and add too much complexity to the code just for one set of docs. I'm not saying never, but at least for now I have other priorities, sorry.

**Comment by kartsims** [comment]
Not so useless, at least we would have access to the HTML code and CSS classes... Just saying ;-)

**Comment by skylarmt** [comment]
I had my work internet go down unexpectedly this afternoon while working on a Bootstrap, HTML5, PHP, and jQuery project.  I was able to lookup everything I needed from DevDocs, except Bootstrap. I was very lucky I happened to have a tab opened to the page I needed to reference.  If I had needed something from their other docs, I would have been SOL.

I know what everything looks like with Bootstrap (heck, half the Internet seems to use it, everyone should know), but the reference has the code.  I don't need the results, if I wanted to see it I could copy-paste the example code into my project (which I do half the time anyway, then edit).
